lua/bulk_endpoints.lua

```diff
@@ -14,9 +14,18 @@ json.strictTypes = true
 local function kick_off_requests_for_bulk_request(endpoint_ids, separator, request_info, cacheability_info)
     local futures = {}
 
+    local pattern = ''
+    local cache_name = ''
+    if cacheability_info ~= nil then
```

Comment:
Are the nil checks for cacheability_info necessary? It doesn't hurt.. but I'm wondering if we can reduce the complexity of the code a bit

cc @drolando

Reply:
If the request is not cacheable we should never get here, so they're probably overkill. If we do get here, then there's a bug somewhere else in the code

Reply:
I would say these are necessary, as `safe get` methods for any hierarchical table should check nullability from the top level. IIRC, a test case or two fails because a `nil` cacheability_info object was passed in.

Reply:
As @drolando and I discussed, I've removed the safe access checks on `cacheability_info`. In the future, should we have the bandwidth, we should implement 500 status error handling, ie `cacheability_info` being nil should throw 500 Internal Server.